### DISCUSSION TITLE
Test help

### DISCUSSION THREAD

**Discussion by vinRosso** [mkdocstrings/python#105] [Q&A] [answered]
Hi, I created a branch to assess and resolve the [issue #158](https://github.com/mkdocstrings/griffe/issues/158) based on my personal test.

The point is that I'm really more of an amateur programmer, so I don't know how to proceed in order to check wether or not it is working. How do I make a custom test to do it?

**Comment by pawamoy** [comment, marked as answer]
Look at the `tests` folder, it contains test functions that you can get inspiration from. I'm not sure what you're really asking though!

**Reply by vinRosso** [reply]
Thanks, I made a custom test and it all went fine!
How do I proceed to push the commits? It says that I don't have the permits to do so.

**Reply by pawamoy** [reply]
I'm not sure to follow: are you in the process of sending a pull request to mkdocstrings-python to allow specifying identifier like the following? (issue https://github.com/mkdocstrings/griffe/issues/158)

```
::: photon/ph_diskconnection/1.6.0/ph_diskconnection.managers.usd.assemblymanager
```

If so, please hold. I'd like to discuss it first. Please open a new issue describing what you would like, and what implementation you suggest.
